src/test/java/org/minions/devfund/marylin/SquarelotronTest.java

```diff
@@ -0,0 +1,557 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * SquarelotronTest class.
+ */
+public class SquarelotronTest {
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static final int THREE = 3;
+    private static final int FOUR = 4;
+    private static final int FIVE = 5;
+
+    //Initialize squarelotrons with different sizes
+    private Squarelotron initialSquarelotronTwoByTwo = new Squarelotron(TWO);
+    private Squarelotron initialSquarelotronThreeByThree = new Squarelotron(THREE);
+    private Squarelotron initialSquarelotronFourByFour = new Squarelotron(FOUR);
+    private Squarelotron initialSquarelotronFiveByFive = new Squarelotron(FIVE);
+
+    /**
+     * Method to print a grid.
+     *
+     * @param grid  grid to print.
+     * @param title title to show.
+     */
+    void printGrid(final int[][] grid, final String title) {
+        System.out.println("========" + title + "=========");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)